src/components/signinForm/SigninForm.js

```diff
@@ -0,0 +1,53 @@
+/* tslint:disable:no-empty */
+import FormControl from '../../shared/FormControl';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import './SigninForm.scss';
+
+
+export class SigninForm extends PureComponent {
+
+	state = {
+		email: '',
+		password: '',
```

Comment:
fix indents

Reply:
This feels pedanctic but its important. Please, fix indents for whole file. 2 spaces. The rules them selves for your code style are not as important as consistency. Try to keep the same feel to each like-expression and like-statements.